Makefile

```
@@ -10,7 +10,7 @@ GOARCH  := $(shell go env GOARCH)
 GOOS    := $(shell go env GOOS)
 GOPROXY := $(shell go env GOPROXY)
 
-GO_FILES=$(shell go list ./... | grep -v /test/sanity)
+GO_FILES=$(shell go list ./... | grep -v /test/bats)
```

```suggestion
GO_FILES=$(shell go list ./...)
```

bats folder has no go files, so we don't need to add that now.

---

Fixed.